frontend/src/metabase/dashboard/components/DashCard/DashCard.styled.tsx

```diff
@@ -41,13 +41,16 @@ export const DashCardRoot = styled.div<DashCardRootProps>`
     shouldForceHiddenBackground && hiddenBackgroundStyle}
 `;
 
-export const DashboardCardActionsPanel = styled.div`
+export const DashboardCardActionsPanel = styled.div<{ leftAlign: boolean }>`
   padding: 0.125em 0.25em;
   position: absolute;
   background: white;
   transform: translateY(-50%);
   top: 0;
-  right: 20px;
+
+  right: ${props => (props.leftAlign ? "initial" : "20px")};
+  left: ${props => (props.leftAlign ? "10px" : "initial")};
```

Comment:
I set it to 10 here, instead of 20, as this case is only for small cards and I feel it looks better and makes it easier to reach for the buttons on the far right